src/main/resources/templates/javascript/readme

```diff
@@ -0,0 +1,5 @@
+#### Tasks
+This is a placeholder exercise.
+
+1. [task][Addition](math should add 1 and 2 to 3)
+2. [task][Multiplication](math should multiply 3 and 5 to 15)
```

Comment:
```suggestion
1. [task][Addition](math should add 1 and 2 to get 3)
2. [task][Multiplication](math should multiply 3 and 5 to get 15)
```

Reply:
Added in https://github.com/ls1intum/Artemis/pull/8859/commits/016767691c11a6d286ee250c493e73b07620d7c7